integrationtests/python_src/px4_it/mavros/missions/avoidance.plan

```diff
@@ -0,0 +1,88 @@
+{
+    "fileType": "Plan",
+    "geoFence": {
+        "circles": [
+        ],
+        "polygons": [
+        ],
+        "version": 2
+    },
+    "groundStation": "QGroundControl",
+    "mission": {
+        "cruiseSpeed": 15,
+        "firmwareType": 12,
+        "hoverSpeed": 5,
+        "items": [
+            {
+                "AMSLAltAboveTerrain": 4,
```

Comment:
I've never had to set this. To help me out, why is this needed? Not suggesting a change. Just curious.

Reply:
Not sure why it was set that way. I downloaded the mission plan I was using to run the test locally. I reverted back to nan.